tests/test_news.py

```diff
@@ -0,0 +1,9 @@
+import json
+
+def test_news(app):
+  client = app.test_client()
+  resp = client.get('/api/v1/news')
+  data = data = json.loads(resp.data.decode())
+  assert resp.status_code == 200
+  assert (data['docs'][0]['featured_image'])
+  assert (data['docs'][0]['title'])
```

Comment:
I just tested that the fields most needed by the FE team are present and that they adhere to the readme for how they will be nested as the news can change each time the test is run.